flame/parallelisms/parallelize_fla.py

```diff
@@ -117,12 +117,12 @@ def apply_tp(
         model,
         tp_mesh,
         {
-            "model.model.embeddings": RowwiseParallel(
+            "model.embeddings": RowwiseParallel(
```

Comment:
The redundant "model" prefix leads to applying RowwiseParallel to embedding layer twice, which is causing the bug described in #11